lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -3026,7 +3026,7 @@ NativeCodeGenerator::GatherCodeGenData(Js::FunctionBody *const topFunctionBody,
 #endif
 
         uint objTypeSpecFldInfoCount = objTypeSpecFldInfoList->Count();
-        jitTimeData->SetGlobalObjTypeSpecFldInfoArray(RecyclerNewArray(recycler, Field(Js::ObjTypeSpecFldInfo*), objTypeSpecFldInfoCount), objTypeSpecFldInfoCount);
+        jitTimeData->SetGlobalObjTypeSpecFldInfoArray(RecyclerNewArray(recycler, Js::ObjTypeSpecFldInfo::ObjTypeSpecFldInfoPtr, objTypeSpecFldInfoCount), objTypeSpecFldInfoCount);
```

Comment:
>Js::ObjTypeSpecFldInfo::ObjTypeSpecFldInfoPtr [](start = 81, length = 45)

Just curious of this change from `Field(Js::ObjTypeSpecFldInfo*)` to a typedef -- is it from a build error or other reason? (I recall had many changes in`NewArray...Field()` pattern.)